src/GeoPing.Api/Controllers/AccountController.cs

```diff
@@ -215,30 +222,55 @@ public IActionResult Register(string returnUrl = null)
         [HttpPost]
         [AllowAnonymous]
         [ValidateAntiForgeryToken]
-        public async Task<IActionResult> Register(RegisterViewModel model, string returnUrl = null)
+        public async Task<IActionResult> Register(RegisterUserDTO registerUser, string returnUrl = null)
```

Comment:
It will not work. add [FromBody] modification for registration model

Reply:
Done